### PR TITLE
feat(interview-prep): Porting Rosetta problems

### PR DESCRIPTION
**Pull request by NitronR**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.org/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [ ] Small bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Addressed currently open issue (replace XXXXX with an issue no in next line)

Closes #XXXXX

### CODE REVIEW

**Comment by scissorsneedfoodtoo** [comment]
@NitronR, thanks for your PR! Sorry for the delay. I'll be reviewing it ASAP.

**Comment by raisedadead** [comment]
@scissorsneedfoodtoo see https://github.com/freeCodeCamp/freeCodeCamp/pull/17471#issuecomment-395914805

File: seed/challenges/08-coding-interview-prep/rosetta-code.json
Comment (scissorsneedfoodtoo) [inline seed/challenges/08-coding-interview-prep/rosetta-code.json]: Could you change this line to the following?:

"contents": ["function gray (enc, number) {", "  // Good luck!", "}"],

File: seed/challenges/08-coding-interview-prep/rosetta-code.json
Comment (scissorsneedfoodtoo) [inline seed/challenges/08-coding-interview-prep/rosetta-code.json]: This one needs a space between the parameters, too:

`"contents": ["function gcd (a,b) {", "  // Good luck!", "}"],`

File: seed/challenges/08-coding-interview-prep/rosetta-code.json
Comment (scissorsneedfoodtoo) [inline seed/challenges/08-coding-interview-prep/rosetta-code.json]: For this challenge, would you mind changing the function name `MaximumSubsequence` to `maximumSubsequence`? Sorry, it's a bit of a pain since they'll have to be changed in the solution, tests, text and text strings, and seed.

I figure this function name should be in camelCase like the rest of them.

**Comment by scissorsneedfoodtoo** [comment]
Just submitted a brief review. Once these changes are made we'll be good to go!

**Review by scissorsneedfoodtoo — CHANGES_REQUESTED** [review]
Sorry for the delay. Once these changes are made we'll be good to go!

**Comment by NitronR** [comment]
@scissorsneedfoodtoo Changes done!! Thanks for the review.

**Comment by scissorsneedfoodtoo** [comment]
@NitronR, alright! Everything looks good, thank you for making those changes so quickly.

@raisedadead, this PR is also ready to merge.

**Comment by scissorsneedfoodtoo** [comment]
@NitronR, just made a couple of minor changes I missed earlier. Just waiting for confirmation before I merge.

**Comment by NitronR** [comment]
OK, thanks!!

**Comment by scissorsneedfoodtoo** [comment]
@NitronR, thank you again for your contribution! :+1: :+1: :+1: 

Everything LGTM.

@raisedadead, this one has been merged.

**Pull request merged**
